app/controllers/cases_controller.rb

```diff
@@ -14,13 +14,16 @@ class CasesController < ApplicationController
                   :edit,
                   :edit_closure,
                   :extend_for_pit,
+                  :extend_sar_deadline,
                   :execute_extend_for_pit,
+                  :execute_extend_sar_deadline,
```

Comment:
think this should be extend_sar_deadline_action for consistency - there are more xxx_action methods than execute_xxx methods